### DISCUSSION TITLE
Can't open using desktop version

### DISCUSSION THREAD

**Discussion by DavTho1983** [dolanmiu/docx#1877] [General]
Hi, I can open a docx I created with our online version of Word, but it won't open on the desktop version and says it's a permissions issue. I've been in touch with our IT support to upgrade ourOffice 365 license and permissions, but I still get the same message.

Any ideas pls?

**Comment by dolanmiu** [comment]
Can you provide a screenshot? It is hard to understand

**Reply by DavTho1983** [reply]
I am on the Microsoft 365 Word launch page and I select a file and choose Open in Desktop App

![Screenshot 2023-01-16 at 10 12 22](https://user-images.githubusercontent.com/13000888/212653604-4a4e81f0-d350-4540-859d-1a57cd98ba85.png)

Then I am directed to the Desktop version of the Word launch page and I click again on the file to open it and I get this message

![Screenshot 2023-01-16 at 10 11 55](https://user-images.githubusercontent.com/13000888/212653758-4c0e3794-a148-40ac-8d87-a5e917e7a515.png)

**Reply by dolanmiu** [reply]
Can you send me some code to reproduce this problem?
